Cargo.toml

```diff
@@ -83,6 +83,8 @@ yubihsm-mock = ["yubihsm/mockhsm"]
 yubihsm-server = ["yubihsm/http-server", "rpassword"]
 fortanixdsm = ["elliptic-curve", "sdkms", "url", "uuid"]
 hashicorp = ["ureq", "base64", "aes-kw", "rsa", "rand", "aes-gcm"]
+default= ["hashicorp"]
```

Comment:
This definitely shouldn't be enabled by default when no other provider is.

Reply:
Oh, sorry... that was mistake